nix/release.nix

```diff
@@ -0,0 +1,7 @@
+# This file wires up the different vaultenv releases that are available.
```

Comment:
There is only one release available, does this comment still apply?

Reply:
Well spotted, no. This PR doesn't use building static binaries, so we only have a single release.